app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -16,6 +16,7 @@ import {
 import type { ReviewIssuesStepValue } from './review-issues-step';
```

Comment:
I followed the testing plan as described in the PR and all worked! Initially I did see the old text using the no_liveness yml on `main`, but after a reset of the server I saw the new text.

Reply:
Thanks! Requiring the server reset is (unfortunately) normal because I'm changing .yml files. Unfortunately, the change [here](https://github.com/18F/identity-idp/pull/10042#discussion_r1495943541) means we need to test again.